patterns/mixed/variables.tf

```diff
@@ -71,12 +65,6 @@ variable "vpcs" {
   }
 }
 
-variable "enable_transit_gateway" {
```

Comment:
Looks like we have this but didn't use it.  We should keep this and fix config.tf to utilize the value instead of hard coding it within.  This goes for each pattern

https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/pre_commit/patterns/mixed/config.tf#L166

Reply:
@akocbek Do you want to make this change then?

Reply:
will update it